src/views/docs/en/reference/configuration/function-config.md

```diff
@@ -193,3 +193,14 @@ Configure Lambda function [CPU `architecture`](https://docs.aws.amazon.com/lambd
 @aws
 architecture arm64
 ```
+
+### `storage`
+
+Configure Lambda function temporary file system size between `512` MB and `10240` MB, in `1` MB increments.
```

Comment:
We should probably mention if you don't specify the storage then it defaults to 512 MB.